Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java

```diff
@@ -584,8 +584,12 @@ public void onViewHolderBind(VPAdapter.ViewHolder holder, int position, Context
                 
     @JavascriptInterface
     public void pauseYoutubeVideo() {
-        if (youTubePlayerView != null) {
-            youtubePlayer.pause();
+        if(vpAdapter != null){
+            vpAdapter.pauseYoutubeVideo();
+            if (youTubePlayerView != null) {
```

Comment:
why we need to stop normal youtube player only if vpadapter present ?

Reply:
there won't be two YouTube player active at once.